rackspace-fabric-faq (#1341)
rackerlabs/support-how-to: content/how-to/rackspace-fabric/rackspace-fabric-faq/index.md

---
permalink: "rackspace-fabric-faq"
audit_date: '2022-05-20'
type: article
title: "Rackspace Fabric FAQ"
created_date: '2022-03-17'
created_by: Asmita Nakwa

> last_modified_date: '2022-05-23'

Reviewer: ```suggestion
last_modified_date: '2022-05-27'
```

---
permalink: "rackspace-fabric-faq"
audit_date: '2022-05-20'
type: article
title: "Rackspace Fabric FAQ"
created_date: '2022-03-17'
created_by: Asmita Nakwa
last_modified_date: '2022-05-23'
last_modified_by: Asmita
product: Rackspace Fabric
product_url: rackspace-fabric
---

### General
{{<accordion title="How can I reset the password of my Email?" col="in" href="accordion1">}}

> If you're a cloud account administrator, go to https://account.rackspace.com/event-history. It should support users in knowing any recent modifications or account access.

Reviewer 2: ```suggestion
If you're a cloud account administrator, go to https://account.rackspace.com/event-history. It should support users to know if there are any recent modifications or account access.
```
Review this in CTUI in Teams channel

{{</accordion>}}

{{<accordion title="How can I return back to my previous ticketing application?" col="in" href="accordion2">}}
Rackspace is deactivating the former ticketing component and migrating clients to Fabric Ticketing, which has been utilized by multi-cloud users for some time. New features and customer feedback enhancements are gradually being integrated in the new Fabric Ticketing application.
{{</accordion>}}

{{<accordion title="I am not able to login to MyRack portal. I have been asked for MFA verification though I have not configured MFA yet." col="in" href="accordion3">}}

> MFA is mandatory for all customers but should allow customer to configure it. In case to remove the MFA please contact identity team. To configure MFA, refer to [Multi-Factor-Authentication](/support/how-to/multi-factor-authentication)

Reviewer 2: ```suggestion
MFA is mandatory for all customers and must be configured by all accounts. To configure MFA, refer to [Multi-Factor-Authentication](/support/how-to/multi-factor-authentication)
```

{{</accordion>}}

{{<accordion title="I am not getting email notification via My Rackportal for incidents pertaining to Alert Logic." col="in" href="accordion4">}}
You will receive the notification if the device in question is enabled in the **Notification Preference** option. To check the preference option, refer to [Notification-Preference](/support/how-to/notification-preference).
 {{</accordion>}}

{{<accordion title="I am unable to login to MyRack Portal. While login it is showing no primary contact exists." col="in" href="accordion5">}}
Create a new ticket in the fabric ticketing app under the category **Account** and select **User Management** as the account type. Contact your Admin or follow up with the support team or Account Manager if you are unable to log into the portal.
{{</accordion>}}

### Permission

{{<accordion title="I am unable to edit a ticket. I am getting the notification as I don't have permission to update." col="in" href="accordion2">}}
Contact an admin (customer side) or contact the Customer Success Manager or Support Team.
{{</accordion>}}

> {{<accordion title="How can I update device name in Vcenter by removing Devices IDs?" col="in" href="accordion5">}}

Reviewer 2: ```suggestion
```

The naming convention for VM is changed. Device names are changed as - DE1CPSxx5 
{{</accordion>}}

{{<accordion title="I am not able to generate PIN via Rackspace portal." col="in" href="accordion6">}}

> According to the instructions here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/), you should be able to unlock the PIN via the portal. If you are unable to do so, you should contact the customer identity team, which is where the PIN is kept. However, bare in mind that this is a security measure. Without a proper verification of the customer's identity, no action should be performed to subvert it.

Reviewer 2: ```suggestion
According to the instructions here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/), you should be able to unlock the PIN via the portal. If you are unable to do so, you should contact the customer support.
```

---
permalink: "rackspace-fabric-faq"
audit_date: '2022-03-17'
type: article
title: "Rackspace Fabric FAQ"
created_date: '2022-03-17'
created_by: Asmita Nakwa
last_modified_date: '2022-03-17'
last_modified_by: Asmita
product: Rackspace Fabric
product_url: rackspace-fabric
---

### General
{{<accordion title="How can I reset the password of my Email?" col="in" href="accordion1">}}

If you're a cloud account administrator, go to https://account.rackspace.com/event-history. It should support users in knowing any recent modifications or account access.
{{</accordion>}}

{{<accordion title="How can I return back to my previous ticketing application?" col="in" href="accordion2">}}
Rackspace is deactivating the former ticketing component and migrating clients to Fabric Ticketing, which has been utilized by multi-cloud users for some time. New features and customer feedback enhancements are gradually being integrated in the new Fabric Ticketing application.
{{</accordion>}}

{{<accordion title="I am not able to login to MyRack portal. I have been asked for MFA verification though I have not configured MFA yet." col="in" href="accordion3">}}

> MFA is mandatory for all customers but should allow customer to configure it. In case to remove the MFA please contact identity team. To configure MFA, refer to [Multi-Factor Authentication](/support/how-to/multi-factor authentication)

Author: Modified as "If MFA is enabled and customer want to remove that, then that should be done by Identity team."

{{</accordion>}}

{{<accordion title="I am not getting email notification via My Rackportal for incidents pertaining to Alert Logic." col="in" href="accordion4">}}
You will receive the notification if the device in question is enabled in the **Notification Preference** option. To check the preference option, refer to [Notification-Preference](/support/how-to/notification prefernce).
 {{</accordion>}}

> {{<accordion title="I am unable to login to MyRack Portal. While login it is showing no primary contact exists." col="in" href="accordion5">}}

Author: Yes, this two issues are different. One is related to MFA and the other about the primary contact already exists.

Create a new ticket in the fabric ticketing app under the category **Account** and select **User Management** as the account type. Contact your Admin or follow up with the support team or Account Manager if you are unable to log into the portal.
{{</accordion>}}

### Permission

> {{<accordion title="I am unable to upload documents and tried multiple times on multiple tickets." col="in" href="accordion1">}}

Author: Removed..

Opt in to the ticketing beta using the ticketing drop down in the global navigation.
{{</accordion>}}

{{<accordion title="I have raised ticket using Rackspace BETA version of ticketing system, but unable to edit the ticket. I am getting the notification as I don't have permission to update." col="in" href="accordion2">}}

> Customer should raise the ticket in one of those 5 categories (Change/Incident/Account/Billing and payments/Request for information) and select appropriate account and product. 

Reviewer: @asmita32675 you would need to use the "Commit Suggestion" dropdown button and save a commit. Then mark this as resolved.

> Customer should raise the ticket in one of those 5 categories (Change/Incident/Account/Billing and payments/Request for information) and select appropriate account and product. 

Author: Resolved.

Also need to contact admin (customer side) or need to contact their Customer Success Manager or Support Team.
{{</accordion>}}

{{<accordion title="I need to access the Managed Backup option in the portal." col="in" href="accordion3">}}

> To access **View Reports**, you can see documentation linked below. To have access to the Managed Backup feature. https://one.rackspace.com/display/myrackspace/Managed+Backup?searchId=OLV66F2LT

Reviewer: This seems like an answer that would work for Rackers. The link is an internal Rackspace link and not available to customers. If you don't have the answer for this right now, then remove the whole FAQ for now and create a separate Jira item to add it in later.

Author: Yes agreed. I have removed it.

{{</accordion>}}

> {{<accordion title="I am having issue in loading MAR report in the portal." col="in" href="accordion4">}}

Reviewer: The link is only for Rackers and is internal. Is this a Racker FAQ or Customer FAQ? Does it belong on this customer page? Remove if its not intended for customers.

Author: FAQ is deleted, since this belongs to internal FAQ.

Reviewer: @asmita32675 I still see the FAQ here on lines 43 - 47 about the MAR process. So I don't think it was removed.

Author: Deleted FAQ, as suggested.

MAR is generated at Global Data Operations side monthly which is then reported to MyRack. 
Below is a documentation with more details:
[MAR Process](https://one.rackspace.com/display/SMBHAMS/MAR+Processes)
{{</accordion>}}

{{<accordion title="How can I update device name in Vcenter by removing Devices IDs?" col="in" href="accordion5">}}

> The naming convention for vm should be <CoreDeviceId>-<device name>, for these vms seems this pattern is not followed so we observed that deviceid is null in our database.

Reviewer: The convention example is not rendering on page. See if we can fix.

Author: Deleted <CoreDeviceId> as per https://core.rackspace.com/py/ticket/view.pt?ref_no=211209-00950.

{{</accordion>}}

{{<accordion title="I am not able to generate PIN via Rackspace portal." col="in" href="accordion6">}}

> The customer should be able to unlock the PIN from the portal, according to the directions posted here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/) if they cannot, I would advise contacting the customer identity team, as that is where the PIN is stored. Be advised though: This is a security measure.  No action should be taken to subvert it without a suitable verification of the customers identity.

Reviewer: ```suggestion
The customer should be able to unlock the PIN from the portal, according to the directions posted here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/) if they cannot, then it is advised to contact the customer identity team, as that is where the PIN is stored. Be advised though: This is a security measure.  No action should be taken to subvert it without a suitable verification of the customers identity.
```

Reviewer: Is this FAQ for Customers? Seems like the answer is speaking to a Racker.

Author: Updated as "According to the instructions here: [Support Pin Management](https://docs.rackspace.com/support/how-to/support-pin-management/), you should be able to unlock the PIN via the portal. If you are unable to do so, you should contact the customer identity team, which is where the PIN is kept. However, bare in mind that this is a security measure. Without a proper verification of the customer's identity, no action should be performed to subvert it."

---
permalink: "rackspace-fabric-faq"
audit_date: '2022-03-17'
type: article
title: "Rackspace Fabric FAQ"
created_date: '2022-03-17'
created_by: Asmita Nakwa
last_modified_date: '2022-03-17'
last_modified_by: Asmita
product: Rackspace Fabric
product_url: rackspace-fabric
---

### General
{{<accordion title="How can I reset the password of my Email?" col="in" href="accordion1">}}

If this is cloud account admin, they can review the following link (https://account.rackspace.com/event-history). It should help them understand any recent changes/access to their account.
{{</accordion>}}

{{<accordion title="How can I return back to my previous ticketing application?" col="in" href="accordion2">}}

> The customer needs to add the firewall as a device and the **Add VPN User** template will be visible in common requests.

Reviewer: I came up with the following for the answer:
`Rackspace is decommissioning the previous ticketing section and moving customers to the Fabric Ticketing experience which has already been used by multi-cloud users for some time. New features and customer feedback improvements will be implemented in the new Fabric Ticketing application and are currently in progress.`

Author: Added mentioned answer.

{{</accordion>}}

{{<accordion title="I am not able to login to MyRack portal. I have been asked for MFA verification though I have not configured MFA yet." col="in" href="accordion3">}}

> MFA is mandatory for all customers but should allow customer to configure it. In case to remove the MFA please contact identity team.

Reviewer: I like the first sentence of answer. However, I believe we need to also add answer to explain how they can configure MFA. If you haven't met with Punit on this answer, then he can probably assist you. So get with him.

Author: As discussed with Punit, MFA procedure is added.

{{</accordion>}}

{{<accordion title="I am not getting email notification via My Rackportal for incidents pertaining to Alert Logic." col="in" href="accordion4">}}

> The customer will get notification only if the device in question is opted in.{{</accordion>}}

Reviewer: Do we have anywhere to link them for setting up or opting in?
Can we explain how they can fix and get email notifications working?
Khazi Pasha or Kshitij Mehta might be able to assist you with this answer if you don't have anyone to answer it.

Author: index.md file for notification preference will be created and linked to the answer. The admin (customer side) can check the preference whether the email notification is enabled or disabled for that particular device.

{{<accordion title="I am unable to login to MyRack Portal. While login it is showing no primary contact exists." col="in" href="accordion5">}}

> There is no primary contact on the account. That is required for the account to work. I believe it can be set from Core.

Reviewer: Check with Punit on this one if you haven't done so already.
This `I believe it can be set from CORE` line might not be useful to the customer. They don't use CORE. Dedicated users would adjust permissions in MyRack. Cloud users would adjust in Apollo or Account sections. So we want to explain where in the customer portal that a customer can change. We could even say for them to have an Admin from the customer side can update it, or instruct them to follow up with their support or their Account Manager.

Author: Updated as Create a new ticket using fabric ticketing application under the category **Account** and select account type as **User Management**.

{{</accordion>}}

### Permission

> {{<accordion title="I am unable to upload documents and tried multiple times on multiple tickets." col="in" href="accordion1">}}

Reviewer: Was this a user that had problems in the previous MyRack ticketing?

Opt in to the ticketing beta using the ticketing drop down in the global navigation.
{{</accordion>}}

{{<accordion title="I have raised ticket using Rackspace BETA version of ticketing system, but unable to edit the ticket. I am getting the notification as I don't have permission to update." col="in" href="accordion2">}}

> User need to have either Edit or Admin permission on the device that is attached to a ticket in order to update the ticket.

Reviewer: So again, we want to be a little more general and not mention the actual permission names for the security reasons and these docs are public.
If a device is on a ticket, and they don't have permissions to that device, then for sure that is one possible cause.
Another cause is they just might not have Ticketing permissions (specifically the Admin permission). They would might be an observer. An observer can only view tickets and can't update a ticket.
So you would have to figure out how to safely word this.

Reviewer: Either way, they need to contact an Admin from the customers side.
If thats not possible, then they need to contact their Customer Success Manager or support.

Author: Customer should raise the ticket in one of those 5 categories (Change/Incident/Account/Billing and payments/Request for information)  and select appropriate account and product. 
Need to contact admin (customer side) or need to contact their Customer Success Manager or Support Team.

Please confirm this.
